src/objects/object.py

```diff
@@ -26,13 +26,8 @@ def __init__(self, position: Point, scene: 'Scene') -> None:
         # Log creation of the `Object`
         print(self)
 
-    def occupy_field(self, position: Point) -> bool:
-        self.scene.get_objects_by_position(position).append(self)
-        return True
-
-    def free_field(self, position: Point) -> None:
-        self.scene.get_objects_by_position(position).remove(self)
-        return True
+    def get_fields(self, position: Point) -> list['Object']:
```

Comment:
You can use types from `typing` e.g. `List` for consistency.

Reply:
Done